src/components/App.jsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
-import { Switch, Route } from 'react-router-dom';
-import './style.css';
-import { NavBar, AllProducts } from '.';
+import React, { useState } from "react";
+import { Switch, Route } from "react-router-dom";
+import "./style.css";
+import { NavBar, AllProducts, SingleProduct, Login, Register } from ".";
+
+// export const [products, setProducts] = useState([]);
+
+// useEffect(() => {
+// 	setProducts().then((products) => {
+// 		setProducts(products);
+// 	})
+// },[])
 
 const App = () => {
-	return (
-		<>
-			<nav>
-				<NavBar currentUser />
-			</nav>
-			<div className="App">
-				<h1>Welcome to Grace Shopper</h1>
-				<h2>The place to buy your products!</h2>
-				<Switch>
-					<Route path="/products">
-						<AllProducts />
-					</Route>
-				</Switch>
-			</div>
-		</>
-	);
+    const [user, setUser] = useState("");
+    return (
+        <>
+            <nav>
+                <NavBar currentUser />
```

Comment:
probably need `currentUser={currentUser}` (although I'm curious what this does). 

using the property like this is, is more common with booleans, i.e. `<Button isDisabled>Submit</Button>` instead of ` <Button isDisabled={true}>Submit</Button>`